tests/kafkatest/tests/client/truncation_test.py

```diff
@@ -34,7 +34,7 @@ class TruncationTest(VerifiableConsumerTest):
 
     def __init__(self, test_context):
         super(TruncationTest, self).__init__(test_context, num_consumers=1, num_producers=1,
-                                             num_zk=1, num_brokers=3, topics=self.TOPICS)
+                                             num_zk=0, num_brokers=3, topics=self.TOPICS)
```

Comment:
sorry that we use `num_zk` to define the `controller_num_nodes_override` so we don't use `0`